lib/index.js

```diff
@@ -218,7 +218,9 @@ function Postgres(a, b) {
 
   function createConnection(options) {
     slots--
-    const connection = Connection(options)
+    // The options object gets cloned as the as the authentication in the frontend.js mutates the
+    // options to persist a nonce and signature, which are unique per connection.
```

Comment:
There's not a single comment in this file. Should I remove it again? 😁

Reply:
I think that's fine considered the non obvious behavior. Maybe we can look at a fix keeping the nonce internal to the connection later to avoid mutating the options.